agent/api/ecsclient/client.go

```diff
@@ -191,11 +201,40 @@ func (client *APIECSClient) registerContainerInstance(clusterRef string, contain
 
 	resp, err := client.standardClient.RegisterContainerInstance(&registerRequest)
 	if err != nil {
-		log.Error("Could not register", "err", err)
+		log.Error("Could not register", "error", err)
```

Comment:
Alright. We've been pushing for any new code additions to use `seelog` instead of `logger`. Up to you.